test/04-verifying/29-sig-geterror.js

```diff
@@ -0,0 +1,28 @@
+import test from 'ava';
+
+var OpenDKIM = require('../../');
+var Messages = require('../fixtures/messages');
+
+var messages = new Messages();
+
+test('test sig_geterror', t => {
+  try {
+    var opendkim = new OpenDKIM();
+
+    opendkim.query_method('DKIM_QUERY_FILE');
+    opendkim.query_info('../fixtures/testkeys');
+
+    opendkim.verify({id: undefined});
+    opendkim.chunk({
+      message: messages.good,
+      length: messages.good.length
+    });
+    opendkim.chunk_end();
+    var error = opendkim.sig_geterror();
+    t.is(error, 0);
+  } catch (err) {
+    console.log(err);
+    t.fail();
+  }
+});
```

Comment:
This test looks good, but we should add one or two more tests for known error conditions.  The same test with:
```
messages.bad_sigalg
messages.bad_signature_version
```
would tell us more about the return value from this function.  That is, that it returns a numeric code on error.